exporter/importables.go

```diff
@@ -1971,4 +2022,75 @@ var resourcesMap map[string]importable = map[string]importable{
 			// {Path: "http_url_spec.authorization", Variable: true},
 		},
 	},
+	"databricks_access_control_rule_set": {
+		AccountLevel: true,
+		Service:      "access",
+		// Name: func(ic *importContext, d *schema.ResourceData) string {
+		// 	return "webhook_" + d.Id()
+		// },
+		List: func(ic *importContext) error {
+			accountId := ic.Client.Config.AccountID
+			// emit default ruleset
+			ic.Emit(&resource{
+				Resource: "databricks_access_control_rule_set",
+				ID:       fmt.Sprintf("accounts/%s/ruleSets/default", accountId),
+			})
+			return nil
+		},
+		Import: func(ic *importContext, r *resource) error {
+			var rule iam.RuleSetResponse
+			s := ic.Resources["databricks_access_control_rule_set"].Schema
+			common.DataToStructPointer(r.Data, s, &rule)
+
+			for _, grant := range rule.GrantRules {
+				for _, principal := range grant.Principals {
+					parts := strings.Split(principal, "/")
+					if len(parts) != 2 {
+						log.Printf("[WARN] Incorrect principal: '%s'", principal)
+						continue
+					}
+					switch parts[0] {
+					case "users":
+						ic.Emit(&resource{
+							Resource:  "databricks_user",
+							Attribute: "user_name",
+							Value:     parts[1],
```

Comment:
Are there any checks we can put on Value? and throw error if not met?

Reply:
Multiple checks are done inside the Emit function (if the service is enabled, etc.), and ImportResource - if a resource is already imported, if a resource exists, etc. It's better to make these checks in a single place (resource itself) instead of checking in each place that is doing E.